doc/guides/test/twister.rst

```diff
@@ -634,3 +634,39 @@ using an external J-Link probe.  The "probe_id" keyword overrides the
       product: DAPLink CMSIS-DAP
       runner: jlink
       serial: null
+
+Quarantine
++++++++++++++++++++++++++++
```

Comment:
make this match length of title and add space below.

Reply:
fixed